packages/synapse-react-client/src/components/EntityFinder/details/view/table/AddToDownloadListCell.tsx

```diff
@@ -0,0 +1,45 @@
+import { IconButton, Tooltip } from '@mui/material'
+import { EntityType } from '@sage-bionetworks/synapse-types'
+import { CellContext } from '@tanstack/react-table'
+import useGetEntityBundle from '../../../../../synapse-queries/entity/useEntityBundle'
+import { useAddFileToDownloadList } from '../../../../../synapse-queries/index'
+import { useSynapseContext } from '../../../../../utils/index'
+import { displayFilesWereAddedToDownloadListSuccess } from '../../../../download_list/DownloadConfirmationUtils'
+import IconSvg from '../../../../IconSvg/IconSvg'
+import { displayToast } from '../../../../ToastMessage/index'
+import { EntityFinderTableViewRowData } from '../DetailsView'
+
+export function AddFileToDownloadListCell(
```

Comment:
All of these cell renderers were moved into their own files to keep them simple & optimized for HMR.

They were changed to use the TanStack table API (which TypeScript will catch), but otherwise should not have many changes, so I don't think they need to be closely reviewed.